docs/security/trust/content_trust.md

```diff
@@ -43,7 +43,7 @@ automated processes such as builds.
 An individual image record has the following identifier:
 
 ```
-[REGISTRY_HOST[:REGISTRY_PORT]/]REPOSITORY[:TAG]
+[REGISTRY_HOST[:REGISTRY_PORT]]/REPOSITORY[:TAG]
```

Comment:
I might be wrong, but does the original one seems to be right?
original one example:
```
REPOSITORY[:TAG]
REGISTRY_HOST/REPOSITORY[:TAG]
REGISTRY_HOST:REGISTRY_PORT/REPOSITORY[:TAG]
```
You changes accepts:
```
/REPOSITORY[:TAG]
REGISTRY_HOST/REPOSITORY[:TAG]
REGISTRY_HOST:REGISTRY_PORT/REPOSITORY[:TAG]
```
Note the first one, is this really what you want?
I'm not familiar with this, just a quick look. :)